<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDialog</class>
 <widget class="QDialog" name="VideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程回放</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_video" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_tool" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_puase">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/img_puse.png</normaloff>:/images/img_puse.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_info" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>190</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px;</string>
        </property>
        <widget class="QLabel" name="lab_coursename">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>25</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>测试</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lab_icon">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>70</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/images/img_history.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>100</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>授课</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="lab_name">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>75</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>多多读</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>72</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>课程列表</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>200</width>
           <height>1</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(218, 218, 218);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_item" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">border: 0px;</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="wgt_videoItem">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>200</width>
              <height>510</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
